docs/integrations/sources/postgres.md

```diff
@@ -172,7 +172,7 @@ Airbyte uses [logical replication](https://www.postgresql.org/docs/10/logical-re
 ### CDC Considerations
 
 - Incremental sync is only supported for tables with primary keys. For tables without primary keys, use [Full Refresh sync](https://docs.airbyte.com/understanding-airbyte/connections/full-refresh-overwrite).
-- Data must be in tables and not views.
+- Data must be in tables and not views. If you require data synchronization from a view, you would need to create a new standard connection.
```

Comment:
Will users know what we mean by "standard connection"?

Reply:
Good point! I'm going to be more explicit there 😄